packages/react-native/src/primitives/Carousel/Carousel.tsx

```diff
@@ -68,7 +69,7 @@ export default function Carousel<T>(
     // Clean up listener. Dimensions.removeEventListener is deprecated as of React Native 0.65 but it is technically
     // available so try to remove via a `EmitterSubscription` first before falling back to `removeEventListener`
     return () => {
-      if (typeof subscription?.remove === 'function') {
+      if (isFunction(subscription?.remove)) {
         subscription?.remove();
```

Comment:
smol nit: is the `?` necessary here if we already check above?

Reply:
Good catch, will update in a follow up PR